readme.md

```diff
@@ -37,3 +37,24 @@ other - number of selected stars should change correctly.
 ---
 ![screenshot](./references/card-example.png)
 
+
+## Завдання
+Створіть сторінки з карткою продукту за допомогою `flexbox` на основі [`the mockup`]
+
+### Вимоги:
+- скинути стандартні поля браузера
+- ширина картки становить 200 пікселів разом із рамкою
+- використовувати зображення з [src/images](src/images)
+- змінити стилі посилань на `:hover`
+- дотримуйтесь стилів з макету
+- додати атрибут `data-qa="card"` в блок картки
+- додати атрибут `data-qa="hover"` до посилання
+
+### Поради та підказки
+- Додайте **ALL** `data-qa` атрибути, необхідні для завдання
+- Повторне використання блоку `зірки` із [завдання Stars](https://github.com/mate-academy/layout_stars)
+і підтримувати його в робочому стані. Якщо ви зміните модифікатор `зірки--4` на `зірки--2` або
+інше - кількість вибраних зірок має змінитися правильно.
+- У вашому коді `card` і `stars` має бути лише 2 блоки BEM, усі інші є лише елементами картки
+- Позиція з текстом «Купити» має бути посиланням.
+- Перевірте стилі шрифтів на макеті. Використовуйте [шрифти Google](https://fonts.google.com/)
```

Comment:
Don't commit any translations. You should change only DEMO and TEST REPORT links here.